lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.scss

```diff
@@ -4,13 +4,9 @@
             color: var(--adf-theme-mat-grey-color-a200-dark);
         }
 
-        mat-card-actions {
+        &-cloud-actions {
             display: flex;
-            justify-content: flex-end;
-
-            .mat-button {
```

Comment:
noticed that this wasn't affecting anything

Reply:
Don't know if I understand what you're asking about, but what I meant with my comment is that I've tested the styles that I deleted and noticed that their presence doesn't change anything in the elements they're describing.

Reply:
You're right, I almost reverted you previous fix